drag_and_drop_v2/public/js/drag_and_drop_edit.js

```diff
@@ -456,9 +456,22 @@ function DragAndDropEditBlock(runtime, element, params) {
                             var $el = $(el),
                                 name = $el.find('.item-text').val(),
                                 imageURL = $el.find('.item-image-url').val(),
+                                expandedImageURL,
                                 imageDescription = $el.find('.item-image-description').val();
 
                             if (name.length > 0 || imageURL.length > 0) {
+                                if (imageURL) {
+                                    var handlerUrl = runtime.handlerUrl(element, 'expand_static_url');
+                                    $.ajax({
+                                        type: 'POST',
+                                        url: handlerUrl,
+                                        data: JSON.stringify(imageURL),
+                                        success: function (result) {
+                                            expandedImageURL = result.url;
+                                        },
+                                        async: false
```

Comment:
I really don't like this synchronous call... If there were ten draggables with images this would be a very bad user experience, since there would be ten blocking network calls in sequence.

Can you not just submit the `imageURL` here? Why do you need to expand the URL at this point?  (Since there's no preview being shown, and `expandedImageURL` is a derived property that doesn't need to be stored, and the Python `submit` handler could compute the `expandedImageURL` instantly if it were somehow necessary).